add docs supporting importing JLCPCB components using RunFrame

Anshgrover23 opened the pull request:
/claim #40
Fixes #40 


https://github.com/user-attachments/assets/ff0f66f5-5d2c-4da2-bc70-2146738cfabb

supporting youtube video link: https://youtu.be/tFCGAa81KUs

imrishabh18 reviewed (commented):
Looks okay to me, Also we need to upload that video from the tscircuit channel and use that YouTube video id.

Anshgrover23 commented:
@imrishabh18 yes, we can just remove my youtube video link from there and upload this video to tscircuit channel as i don't have perms for that, i think @seveibar can do that.

seveibar commented:
I think linking to ansh’s youtube is ok for now, and the video is good, but i do think it would be better if you then wrote the import in the index file so that the user could see that the import works and is usable

seveibar commented on package.json:
```diff
@@ -16,6 +16,7 @@
     "typecheck": "tsc"
   },
   "dependencies": {
+    "@devbookhq/docusaurus-code-video-plugin": "^1.2.1",
```
Why?

Anshgrover23 replied:
removed.

seveibar reviewed (changes requested):
End users dont know what runframe is, you should explain as “from tsci dev” or “using the cli” in the doc text

Anshgrover23 commented:
@seveibar here is the new video addressing your comment of importing into index.ts file included in video.


https://github.com/user-attachments/assets/a23da1a5-686b-4155-adcb-6af94b490078

check the new video in deployement.

Anshgrover23 committed:
change youtube video link

seveibar reviewed (approved):
Hey Ansh, this is quite good. I would have used a more complex chip though and given it the name U1. It's not really clear what you imported because it isn't a common chip AFAIK. Importing something like the PICO_W would be really good and have a complex symbol. I would also remove the other components from the design (the trace resistor and capacitor) because they're clutter than confuses the main point. Feel free to follow up if you'd like, but approving/merging because this is an improvement over the existing docs.
